cmd/json.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/DisposaBoy/JsonConfigReader"
+)
+
+func ParseJSONWithComments(input []byte, obj interface{}) error {
+	r := JsonConfigReader.New(bytes.NewReader(input))
+	return json.NewDecoder(r).Decode(obj)
+}
+
+func ConvertToJsonSafe(val interface{}) interface{} {
+	switch v := val.(type) {
+	case map[interface{}]interface{}:
+		//valMap :=
+		res := map[string]interface{}{}
+		for k, v := range v {
+			switch v2 := v.(type) {
+			case map[interface{}]interface{}, []interface{}:
```

Comment:
isn't it enough res[k] = ConvertToJsonSafe(v)?

Reply:
probably 			res[fmt.Sprint(k)] = convertToJSONSafe(v) will be ok - changed